snapcraft/_store.py

```diff
@@ -381,9 +381,16 @@ def sign_build(snap_filename, key_name=None, local=False):
             'Build assertion {} pushed to the Store.'.format(snap_build_path))
 
 
-def update_metadata(snap_yaml, force):
-    """Update metadata in the server."""
+def push_metadata(snap_filename, force):
+    """Push only the metadata to the server.
+
+    If force=True it will force the local metadata into the Store,
+    ignoring any possible conflict.
+    """
+    logger.info("Pushing metadata to the Store (force=%s)", force)
```

Comment:
Mind removing this line or making it `.debug` or we will get a double print on regular runs?